outputs.tf

```diff
@@ -8,6 +8,16 @@ output "subnets" {
   description = "map ||| A map with keys of form subnet_region/subnet_name and values being the outputs of the google_compute_subnetwork resources used to create corresponding subnets."
 }
 
+output "subnets_names" {
```

Comment:
In both `subnets_names` and `subnets_ids`, it would be great to delineate between the types of subnets.
See https://github.com/nullstone-modules/aws-network/blob/master/outputs.tf#L11-L39 for an example.

This allows modules to place appliances in logical network zones.
Load balancers go in `public`, services go in `private`, databases in `private` or `intra` depending on preference/ability to operate.